core/src/main/scala/io/snappydata/Literals.scala

```diff
@@ -274,6 +274,14 @@ object Property extends Enumeration {
         "scalability of queries in the interest of reduced memory usage for " +
         "secondary buckets. Default is false.", Some(false), Constant.SPARK_PREFIX)
 
+  val PlanCaching: SQLValue[Boolean] = SQLVal[Boolean](
+    s"${Constant.PROPERTY_PREFIX}planCaching",
+    "Property to set/unset plan caching", Some(true))
+
+  val PlanCachingAll: SQLValue[Boolean] = SQLVal[Boolean](
+    s"${Constant.PROPERTY_PREFIX}planCachingAll",
+    "Property to set/unset plan caching on all sessions", Some(true))
```

Comment:
I think its better to use proper qualified name like spark does for all its properties. For example: snappydata.sql.planCaching and snappydata.sql.planCachingAll

Reply:
ok. I don't thnk there is any existing property like this put in sql from our side. Will add sql.